### PR TITLE
Changes for failed tests in Finsemble.

### PR DESCRIPTION
**Pull request by gaganahluwalia**

### CODE REVIEW

**Review by kriswest — APPROVED** [review]
LGTM

File: directories/local-conformance-2_0.finsemble.apps.json [hunk 1]
```diff
@@ -1,885 +1,940 @@
 {
-  "comment": "AppD component config",
```
Comment (robmoffat) [inline directories/local-conformance-2_0.finsemble.apps.json:2]: this file shouldn't have finsemble in the name - Finsemble uses the standard appD record now.

File: src/mock/v2.0/intent-l.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { closeWindowOnCompletion, onFdc3Ready } from "./mock-functions";
```
Comment (robmoffat) [inline src/mock/v2.0/intent-l.ts:1]: this shouldn't be intent-L, the intent-* mocks are for the Intents tests.  This is a basic test, so maybe call it "basic"?

File: src/test/v2.0/support/intent-support-2.0.ts [hunk 3]
```diff
@@ -102,7 +102,7 @@ export class RaiseIntentControl2_0 {
   }
 
   validateIntentResult(intentResult, expectedIntentResultType: IntentResultType, expectedContextType?: string) {
-    expect(typeof intentResult).to.be.equal("object");
+    expect(typeof intentResult).to.be.oneOf(["object", 'undefined']);
```
Comment (robmoffat) [inline src/test/v2.0/support/intent-support-2.0.ts:105]: remove this  - it's taken care of later.

File: webpack.config.js [hunk 4]
```diff
@@ -26,6 +26,7 @@ const config = {
     "intent-i-v2.0": "./src/mock/v2.0/intent-i.ts",
     "intent-j-v2.0": "./src/mock/v2.0/intent-j.ts",
     "intent-k-v2.0": "./src/mock/v2.0/intent-k.ts",
+    "intent-l-v2.0": "./src/mock/v2.0/intent-l.ts",
```
Comment (robmoffat) [inline webpack.config.js:29]: rename here

**Review by robmoffat — CHANGES_REQUESTED** [review]

**Review by robmoffat — APPROVED** [review]

**Pull request merged**